CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
     * Add `CardPresentAuthChallengeResult` type
     * Add `CardClient.completeAuthChallenge(Intent, String)` method
     * Add `CardStatus` type
+    * Add `ApproveOrderListener.onApproveOrderAuthorizationRequired(CardAuthChallenge)` method
```

Comment:
Isn't it renaming?

Reply:
It is technically a rename, though the entire concept of this method is new in `v2`. Since we've never released it I chose to put "add" in the CHANGELOG.